downloads/swan-lake-release-notes/swan-lake-2201.7.0/RELEASE_NOTE.md

```diff
@@ -0,0 +1,237 @@
+---
+layout: ballerina-left-nav-release-notes
+title: 2201.7.0 (Swan Lake) 
+permalink: /downloads/swan-lake-release-notes/2201.7.0/
+active: 2201.7.0
+redirect_from: 
+    - /downloads/swan-lake-release-notes/2201.7.0
+    - /downloads/swan-lake-release-notes/2201.7.0-swan-lake/
+    - /downloads/swan-lake-release-notes/2201.7.0-swan-lake
+    - /downloads/swan-lake-release-notes/
+    - /downloads/swan-lake-release-notes
+---
+
+## Overview of Ballerina Swan Lake 2201.7.0
+
+<em>2201.7.0 (Swan Lake Update 7) is the seventh update release of Ballerina Swan Lake, and it includes a new set of features and significant improvements to the compiler, runtime, standard library, and developer tooling. It is based on the 2023R1 version of the Language Specification.</em> 
+
+## Update Ballerina
+
+Update your current Ballerina installation directly to 2201.7.0 using the [Ballerina Update Tool](/learn/cli-documentation/update-tool/) as follows.
+
+1. Run `bal update` to get the latest version of the Update Tool.
+2. Run `bal dist update` to update to this latest distribution.
+
+## Install Ballerina
+
+If you have not installed Ballerina, download the [installers](/downloads/#swanlake) to install.
+
+## Backward-incompatible changes
+
+- A bug that allowed using a function reference of a `non-isolated` function type in a function or method call expression within an `isolated` function or method has been fixed.
```

Comment:
```suggestion
- A bug that allowed using a function reference of a non-`isolated` function type in a function or method call expression within an `isolated` function or method has been fixed.
```

Same comment re: making changes to already approved content - https://github.com/ballerina-platform/ballerina-dev-website/pull/7156/files#diff-a2d9e6006fef3a642196e95b059536ee565b283e4dbdeaa8b22bd3892b4104c9R31.